File: src/code/Inscriptions.java
package code;
import java.util.ArrayList;
import java.util.List;


public class Inscriptions {
    private List<Equipe> equipes = new ArrayList<Equipe> ();

    private Tournoi tournoi;

    private List<Inscriptions> inscriptions = new ArrayList<Inscriptions> ();

    private Inscriptions(Tournoi tournoi) {
    }

    public void ajouterEquipe(Equipe equipe) {
    }

    public List<Equipe> getEquipes() {
        return equipes;
    }

    public static Inscriptions getInstance(Tournoi tournoi) {
        return null;
    }

    public String toString() {
        return null;
    }

    public Tournoi getTournoi() {
        return tournoi;
    }

}

File: modele/Inscriptions.php
<?php
//créer une inscription
class Inscriptions
{
    private $equipes = array();
    private $tournoi;
    private $inscriptions = array();
    //constructeur
    private function __construct($tournoi){
        $this->equipes = NULL;
        $this->tournoi = $tournoi;
    }
    //ajouter une équipe dans inscrit
    public function ajouterEquipe($equipe)
    {
    }
    //récupéré les équipes
    public function getEquipes()
    {
        return $this->equipes;
    }
    //récupéré l'instance d'un tournoi
    public static function getInstance($tournoi)
    {
        return NULL;
    }
    //to string
    public function toString()
    {
        return NULL;
    }
    //récupéré le tournoi
    public function getTournoi()
    {
        return $this->tournoi;
    }
}
?>